src/main/java/com/team4/testingsystem/controllers/ErrorReportsController.java

```diff
@@ -1,25 +1,42 @@
 package com.team4.testingsystem.controllers;
 
+import com.team4.testingsystem.converters.ErrorReportsConverter;
 import com.team4.testingsystem.dto.ErrorReportDTO;
 import com.team4.testingsystem.services.ErrorReportsService;
 import io.swagger.annotations.ApiOperation;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @RestController
 @RequestMapping(path = "/error_reports")
 public class ErrorReportsController {
 
     private final ErrorReportsService errorReportsService;
+    private final ErrorReportsConverter errorReportsConverter;
 
     @Autowired
-    public ErrorReportsController(ErrorReportsService errorReportsService) {
+    public ErrorReportsController(ErrorReportsService errorReportsService,
+                                  ErrorReportsConverter errorReportsConverter) {
         this.errorReportsService = errorReportsService;
+        this.errorReportsConverter = errorReportsConverter;
+    }
+
+    @ApiOperation(value = "get a list of error reports by test id")
+    @GetMapping(path = "/{testId}")
+    public List<ErrorReportDTO> getByTestId(@PathVariable long testId) {
```

Comment:
Why do we really need a separate endpoint for it?

Reply:
It was necessary to get a list of error reports. I thought that it meant getting them by the test id. By what parameter should I have got a list of error reports?

Reply:
Put the data to a corresponding dto. We don't need to have a separate endpoint for it